### ISSUE TITLE
the number of samples

### ISSUE THREAD

**Issue by youmi-zym** [ClementPinard/SfmLearner-Pytorch#29]
Hi,
    Thanks for your implementation in PyTorch, it helps me much,  recently I learn the ego-motion with your code. The procession of data preparation is quiet rambling for me, and I'm quiet uncertain whether there are some errors I have made. Therefore, can you tell me the final number of samples in **KITTI** and **Cityscapes**? 
For me, I got
KITTI:
43154 samples found in 64 train scenes
1090 samples found in 8 valid scenes

**Comment by youmi-zym** [comment]
3 frames...
-with-depth --with pose --static-frames=/the/file/you/provide/

And how about Cityscapes? I have not got yet....

**Comment by youmi-zym** [comment]
@ClementPinard  and --static-frames=/the/file/you/provide/, the output is 
43310 samples found in 64 train scenes
934 samples found in 8 valid scenes

**Comment by ClementPinard** [comment]
The algorithm chooses 8 scenes at random to constitute the validation set, without looking at the number of frames in each scene, so it's competely normal to have different amount of samples.
However, the total amount of frames should be the same.
You can see that mine is not the same because I didn't use the static frames file that was originally provided by author, but the condition on car speed. (namely frame must be at the equivalent of 2 m.s apart, which means for 10fps a translation between frames of at least 20cm)

**Comment by youmi-zym** [comment]
@ClementPinard Thanks for your detailed reply, I got it.
